include/swift/Driver/DependencyGraph.h

```diff
@@ -123,9 +123,17 @@ class DependencyGraphImpl {
   /// The set of marked nodes.
   llvm::SmallPtrSet<const void *, 16> Marked;
 
-  /// A list of all external dependencies that cannot be resolved
-  /// just from this dependency graph. They must be resolved from definitions
-  /// in files that do not belong to this module.
+  /// A list of all external dependencies that cannot be resolved from just this
+  /// dependency graph. Each member of the set is the name of a file which is
+  /// not in the module. These files' contents may (or may not) have affected
+  /// the module's compilation. This list may even include the paths of
+  /// non-existent files whose absence is significant in the same way as
+  /// Makefile .d dependencies.
```

Comment:
This "in the same way as Makefile .d dependencies" is ambiguously attached to the rest of the sentence in a way that makes it more confusing. If it's not helping you, just leave it out; otherwise please rephrase.

Reply:
Good catch! Removed.